README.md

```diff
@@ -15,4 +17,13 @@ Python data structures.
 $ pip install cbor-decoder
 ```
 
+## Run tests
+
+```
+$ pip install -U pytest
+$ pytest
```

Comment:
After installing there was no `pytest` command for me. Is there some other way to install or run the tests?